tests/org.eclipse.ui.workbench.texteditor.tests/src/org/eclipse/ui/workbench/texteditor/tests/FindReplaceDialogTest.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2000, 2016 IBM Corporation and others.
+ * Copyright (c) 2024 Vector Informatik GmbH and others.
```

Comment:
Please revert this copyright change.

Reply:
changed!

Reply:
https://github.com/eclipse-platform/eclipse.platform.ui/commit/2f9136a25069244769073ee77ad5be15b5dadeee